src/bin/rdcore/cmdline.rs

```diff
@@ -146,8 +146,8 @@ pub struct ZiplConfig {
     pub hostkey: Option<String>,
 
     /// Extra kargs
```

Comment:
Should we change this to e.g. `Append kernel argument` for clarity and consistency with `rdcore kargs`?